mysql-test/suite/galera/t/mysql-wsrep-372.test

```diff
@@ -0,0 +1,315 @@
+#
```

Comment:
The same test in 5.7 has the name `mysql-test/suite/galera/t/mysql-wsrep#372.test`. Can you please rename the testcase to the same name as 5.7?

Reply:
I used the name codership has in their 8.0 sources, I'm not sure if we should rename it to something different.

Reply:
Okay. Then it is not a problem. Let us leave as is.